pkgs/by-name/av/avalanchego/package.nix

```
@@ -1,26 +1,27 @@
-{ IOKit
-, buildGoModule
-, fetchFromGitHub
-, lib
-, stdenv
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+  nix-update-script,
+  stdenv,
 }:
 
 buildGoModule rec {
   pname = "avalanchego";
-  version = "1.11.11";
+  version = "1.12.0-initial-poc.6";
 
   src = fetchFromGitHub {
     owner = "ava-labs";
     repo = pname;
     rev = "v${version}";
```

```suggestion
    repo = "avalanchego";
    rev = "refs/tags/v${version}";
```
Sorry, didn't see this first time around. https://github.com/nix-community/nixpkgs-lint/issues/21